private/testplanet/satellite.go

```diff
@@ -550,18 +552,23 @@ func (planet *Planet) newSatellite(ctx context.Context, prefix string, index int
 		return nil, err
 	}
 
+	rlPeer, err := planet.newRangedLoop(ctx, index, identity, db, metabaseDB, config)
```

Comment:
Note: for the long term I think it would be good if we don't have different types of peers, but instead have a single type of satellite which gets its responsibilities from the config.